.github/workflows/go.yml

```diff
@@ -0,0 +1,75 @@
+name: Go
+
+on:
+  push:
+    branches: ['*']
+    tags: ['v*']
+  pull_request:
+    branches: ['*']
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Setup Go
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.14.x
```

Comment:
(no action for this change) if we wanted to test multiple versions (not important for yab, since it's not a library), can we easily do that?

Reply:
Yeah, there's support for multiple versions. i have that in one of my repos.